### PR TITLE
Create Bug/Feature report and PR template

### PR DESCRIPTION
**Pull request by m273d15**
In order to induce people to provide more detailed reports I created a bug and a feature report template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Requesting that the user attach the log files (Saros log and possibly IDE log) of a session in which the error appeared would be very important in my opinion.

**Comment by m273d15** [comment]
@tobous Yes, this was the idea.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (m273d15) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Where can I find the IntelliJ logs and the Saros/I logs?

Comment (tobous) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The different log locations are described [here](https://www.saros-project.org/releases/saros-i_0.1.0.html#report-a-bug).

Comment (m273d15) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I added the information, but I am not sure whether the template became a bit crowded with information and we should rather link this information.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+**Log files**
+
+Please provide the corresponding log files, but **think of substituting sensitive data** you don't want to publish.
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please provide the corresponding log files, but be sure to **remove or obfuscate sensitive data** you don't want to publish.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+You can find the log files in the following locations:
+* IntelliJ IDEA (with default config), where `.IdeaXXXXXXXX` is the directory containing the IDE files (e.g. `.IdeaIC2019.1`).</br>
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]: As this is mostly read in the editor and not displayed (as it should be removed by the user), I don't think the formatting (e.g. `<br>` or specified link references `[text](link)`) makes much sense here and just makes it harder to read.

Markings for fields (e.g. using `) is still sensible in my eyes.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  See [here](https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs) for more information about the location of the directory.
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  More information about the location of the directory is given here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  * IDE logs - `~/.IdeaXXXXXXXX/system/log/`
+  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/`.
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/`
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'State: Unconfirmed, Type: Feature'
+assignees: m273d15
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
A clear and concise description of what the problem is. E.g. I'm always frustrated when [...]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Log files**
+
+Please provide the corresponding log files, but be sure to **remove or obfuscate sensitive data** you don't want to publish.
+
+You can find the log files in the following locations:
+* IntelliJ IDEA (with default config), where `.IdeaXXXXXXXX` is the directory containing the IDE files (e.g. `.IdeaIC2019.1`).
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
* IntelliJ IDEA:
  By default, the directory `.IdeaXXXXXXXX` is located in the user directory. The exact directory name is determined by your IntelliJ version, e.g. `.IdeaIC2019.1` for IntelliJ IDEA 2019.1.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+  More information about the location of the directory is given here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs
+  * IDE logs - `~/.IdeaXXXXXXXX/system/log/`
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  * IDE logs - `~/.IdeaXXXXXXXX/system/log/idea.log*`
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/`
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/*.log`
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Log files**
+
+Please provide the corresponding log files, but be sure to **remove or obfuscate sensitive data** you don't want to publish.
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please provide the corresponding log files, but be sure to **remove or obfuscate sensitive data** you don't want to publish. Please also make sure only to include the log files concerning the session you encountered the bug in.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+You can find the log files in the following locations:
+* IntelliJ IDEA (with default config), where `.IdeaXXXXXXXX` is the directory containing the IDE files (e.g. `.IdeaIC2019.1`).
+  More information about the location of the directory is given here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs
+  * IDE logs - `~/.IdeaXXXXXXXX/system/log/`
+  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/`
+* Eclipse:
+  * IDE logs - `<workspace>/.metadata/.log`
```
Comment (tobous) [inline .github/ISSUE_TEMPLATE/bug_report.md]: is the file really named `.log` or should this be `*.log` as well?

Comment (m273d15) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes, the name `.log` is correct

**Pull request merged**